tomviz/EditOperatorDialog.h

```diff
@@ -39,8 +39,26 @@ class EditOperatorDialog : public QDialog
                      bool needToAddOperator, QWidget* parent);
   virtual ~EditOperatorDialog();
 
+  // Used to set the mode of the EditOperatorWidget in the dialog.  The mode
+  // corresponds to dialog options like tabs and varies from operator to
+  // operator.
```

Comment:
Can we fix up the line wrapping in the comments please. Several others are the same, I am guessing an artifact from clang-format - enter as one long line and let it reflow accordingly, or use a static word wrap marker in your code editor at 80 characters to help.

Reply:
I've updated the comments to wrap better.